apple/testing/default_runner/ios_test_runner.template.sh

```diff
@@ -18,33 +18,34 @@ set -e
 
 basename_without_extension() {
   local full_path="$1"
-  local filename=$(basename "$full_path")
+  local filename
+  filename=$(basename "$full_path")
```

Comment:
what's the reason for this difference with the previous variable?

Reply:
Shellcheck validates that you don't define and assign to a variable if the assignment is a command that could also fail https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
(behavior is identical)